.editorconfig

```diff
@@ -2,16 +2,14 @@
 [*]
 
 # ReSharper properties
-resharper_blank_lines_after_start_comment=0
```

Comment:
Why did you remove some rules? They seem useful.

Reply:
Returned some rules.